Flickr/Flickr/ImageCollectionViewController.swift

```diff
@@ -10,17 +10,32 @@
 import UIKit
 import Alamofire
 
-class ImageCollectionViewController: UIViewController, UICollectionViewDelegate, UICollectionViewDataSource, UITextFieldDelegate {
-  
+class ImageCollectionViewController: UIViewController, UICollectionViewDelegate, UICollectionViewDataSource, UITextFieldDelegate, UICollectionViewDelegateFlowLayout {
+    
     fileprivate var imageData: [Photo] = [Photo]()
+    var isSearching = false
+    var sizes: [CGSize] = [CGSize]()
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        searchCollectionView.alpha = 1
+        popularCollectionView.alpha = 0
+        searchCollectionView.delegate = self
+        searchCollectionView.dataSource = self
```

Comment:
why delegate for collection is set only for searchCollectionView? Make this work for popularCollectionView or move delegate set in Interface bulder